app/controllers/forms/ContributionRequest.scala

```diff
@@ -47,4 +49,15 @@ object ContributionRequest {
   implicit val contributionRequestReads: Reads[ContributionRequest] = Json.reads[ContributionRequest]
 }
 
+case class MetadataUpdate(marketingOptIn: Boolean)
+
+object MetadataUpdate {
```

Comment:
Could we replace `metadata` everywhere here with `marketingOptIn` since that's all it does - the word metadata is a bit overloaded and keeps making me wonder whether this is being used to update Stripe/PayPal metadata - which it's not

Reply:
in fact - I'd be inclined to just put this inside `IdentityController` since that's the only place it's used and ever will be used